import-export-cli/integration/apim/apim.go

```diff
@@ -2295,3 +2297,178 @@ func getTokenURL(host string, offset int) string {
 	port := 9443 + offset
 	return "https://" + host + ":" + strconv.Itoa(port) + "/oauth2/token"
 }
+
+// GetAPIPolicies : Get API Policies list of all types from APIM
+func (instance *Client) GetAPIPolicies(t *testing.T, offset, limit string) *APIPoliciesList {
+	var policyListResponse *APIPoliciesList
+
+	getPoliciesURL := instance.publisherRestURL + "/operation-policies"
+
+	queryParams := "offset=" + offset + "&limit=" + limit
+
+	if offset != "-1" && limit != "-1" {
+		getPoliciesURL += "?" + queryParams
+	}
+
+	request := base.CreateGet(getPoliciesURL)
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.GetAPIPolicies()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	base.ValidateAndLogResponse("apim.GetAPIPolicies()", response, 200)
+
+	err := json.NewDecoder(response.Body).Decode(&policyListResponse)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	return policyListResponse
+}
+
+// DeleteAPIPolicy : Deletes API Policy from APIM using UUID
+func (instance *Client) DeleteAPIPolicy(policyID string, baseTest string) {
```

Comment:
```suggestion
func (instance *Client) DeleteAPIPolicy(policyID, baseTest string) {
```
Let's change like this in all the applicable places.

Reply:
Fixed by [59b2698](https://github.com/wso2/product-apim-tooling/pull/925/commits/59b2698c4e9aceab8b212ba2afd761955611f7e4)